### PR TITLE
Dark theme

### PR DESCRIPTION
**Pull request by SuperTux88**
I created a dark theme for diaspora:

diaspora-dark

(If you have an account on nerdpol.ch you can also test it there.)

I also refactored the default theme a bit, to use more color-variables and less overrides (makes diaspora better themeable if you want to change more than the header color). And I renamed the default theme to "Original Gray" instead of "Original Dark" because I found it weird, because the default theme isn't "dark" (only the header is dark).

### CODE REVIEW

**Comment by AugierLe42e** [comment]
Wow... That's... Really dark !

**Comment by goobertron** [comment]
I think 'Default' might be a better name instead of 'Original [Dark/Gray]'. What do others think?

ps: you should make the text and images black as well for a *really* dark theme...

**Comment by SuperTux88** [comment]
> Wow... That's... Really dark !

Yes, thats the intention of a dark theme ;)

> I think 'Default' might be a better name instead of 'Original [Dark/Gray]'. What do others think?

I don't know. Podmins can set another default-theme and I think "Default" as name can be weird then?

> ps: you should make the text and images black as well for a *really* dark theme...

No, I think  it's dark enough ;)

**Comment by AugierLe42e** [comment]
> Yes, thats the intention of a dark theme ;)

Sad. I would really love to have a dark gray theme. But this one misses some contrast for me :(

**Comment by SuperTux88** [comment]
I made the backgrounds a bit darker and the text-color lighter, so it have more contrast now.

screen shot 2016-11-05 at 19 46 16

I also updated nerdpol.ch with the new theme.

File: app/assets/stylesheets/mobile/mobile.scss [hunk 1]
```diff
@@ -176,7 +176,7 @@ footer {
     border-radius: 3px 3px 0 0;
 
     border: {
-      bottom: 1px solid #ccc;
+      bottom: 1px solid $border-medium-grey;
```
Comment (denschub) [inline app/assets/stylesheets/mobile/mobile.scss]: When you are already changing and adding variables... can we try to get a more generic naming? This border may not be grey and I'd rather have a generic name for the variables. Also, we should find a consent between grey and gray, I guess...

**Comment by denschub** [comment]
This is nice.

File: app/assets/stylesheets/mobile/mobile.scss (hunk 1 above)
Comment (SuperTux88) [inline app/assets/stylesheets/mobile/mobile.scss]: `$border-medium-grey` existed before, I'm only using it instead of the hardcoded color.

But I agree with you about the naming. I tried to use better names for new variables I created, but I didn't rename existing variables. I didn't want to change too much here, the PR is already big enough. I only used the existing variables instead of hardcoded colors, so I need less css-hacks here. I think the renaming should be done in another PR?

Comment (denschub) [inline app/assets/stylesheets/mobile/mobile.scss]: > I think the renaming should be done in another PR?

Well. This probably means it will never be done, but fair point. Okay. :)

Comment (SuperTux88) [inline app/assets/stylesheets/mobile/mobile.scss]: I love cleanups, so I'll do it sometime (if nobody else does it), but I'd rather not do it now as drive-by for a theme. ;)

File: app/assets/stylesheets/color_themes/_color_theme_override_dark.scss [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+  #welcome-to-diaspora { background: $orange; }
+
+  .block-form fieldset .form-control:focus { border-color: $input-border; }
+
+  &.page-registrations.action-new,
+  &.page-registrations.action-create {
+    .ball { filter: invert(100%); }
+  }
+
+  .spinner { border-color: $gray-light transparent $gray-light $gray-light; }
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/color_themes/_color_theme_override_dark.scss]: Shorthands of length `4` are not allowed. Value was `$gray-light transparent $gray-light $gray-light`

**Comment by denschub** [comment]
Sweet stuff, merged. Thank you!
